.azure-pipelines/pipeline.yml

```diff
@@ -0,0 +1,40 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish @vscode/iconv-lite-umd
+    type: boolean
+    default: false
+
+extends:
+  template: azure-pipelines/npm-package/pipeline.yml@templates
+  parameters:
+    npmPackages:
+      - name: @vscode/iconv-lite-umd
```

Comment:
@bpasero, can you change the name to `iconv-lite-umd` as the name is used as a "stage name" and "@" is not a valid character. Thanks!